# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mac-fdisk/compile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- mac-fdisk-0.1.orig/io.c.vanilla	2005-05-30 11:35:00.000000000 +0200
+++ mac-fdisk-0.1.orig/io.c	2005-05-30 11:35:33.000000000 +0200
@@ -35,6 +35,7 @@
 #include <string.h>
 #include <stdarg.h>
 #include <errno.h>
+#include <sys/types.h>
 
 #include "pdisk.h"
 #include "io.h"
